src/components/Canvas/FallingBall.jsx

```diff
@@ -0,0 +1,58 @@
+// @ts-check
```

Comment:
Curious: does vim grok this? Do you get real time type checking as you code?

Reply:
I think I'd need to install a plugin - Vim doesn't support type checking out of the box.  Honestly, I should just use VSC for debugging.